.github/workflows/publish-dependency-image.yml

```diff
@@ -0,0 +1,45 @@
+name: Publish Dependency Analysis Docker image
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - 'v*'
+
+env:
+  REGISTRY: ghcr.io
```

Comment:
We can do that. Probably after this PR gets merged in then, we can update it. I would like to get this merged so that we can beta-test it. I agree before going PROD we will update it.